test/framework/src/main/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -2144,4 +2146,11 @@ private static boolean isSuiteScopedTest(Class<?> clazz) {
     public @interface SuppressNetworkMode {
     }
 
+    public static Index resolveIndex(String index) {
+        GetIndexResponse getIndexResponse = client().admin().indices().prepareGetIndex().setIndices(index).get();
+        assertTrue(getIndexResponse.getSettings().containsKey(index));
```

Comment:
I think it'd be nice to add an error message here so it is more clear that this is an "index not found" thing.